react-app/src/features/forms/waiver/app-k/index.test.tsx

```diff
@@ -22,6 +22,13 @@ describe("Appendix K", () => {
     expect(amendmentTitle).toBeInTheDocument();
   });
 
+  test("Waiver number link should have correct capitalization", async () => {
```

Comment:
dev note: if this test seems unnecessary im OK taking it out.

Reply:
You can remove this test because it's tested in `ActionForm.test.tsx`